frontend/src/App.tsx

```diff
@@ -17,6 +19,8 @@ function App() {
         <Route path="/live/:sessionKey/" element={<CurrentSongView />}>
           <Route path="add-song" element={<AddSongDrawer />} />
         </Route>
+        <Route path="/live/createsongbook" element={<CreateNewSongbook />} />
```

Comment:
This is just a temporary path until we find a home for this UX, right? If not, I'd say rename it to `/live/create-songbook` to be consistent

Reply:
Temporary until it's put into a drawer.